pkg/cli/config/initialize/sls/networkBuilder.go

```diff
@@ -78,7 +78,7 @@ Handy Netmask Cheet Sheet
 
 const (
 	// DefaultMTLVlan is the default MTL Bootstrap Vlan - zero (0) represents untagged.
-	DefaultMTLVlan = 0
+	DefaultMTLVlan = 1
```

Comment:
This will cause problems. There is code that compares the MTL VLAN to `0` that causes `bond0` to be the metal network name instead of `bond0.mtl0`.

Reply:
NCNs will be affected by this due to: https://github.com/Cray-HPE/node-images/blob/main/metal-provision/roles/node_images_ncn/files/cloud/templates/cloud-init-network.tmpl#L24